packages/@okta/i18n/src/properties/login.properties

```diff
@@ -718,6 +718,8 @@ oie.password.confirmPasswordLabel = Re-enter password
 oie.password.match.error = New passwords must match
 oie.password.expired.title = Your password has expired
 oie.password.expired.primaryButton = Change Password
+oie.password.expiry.warning.title = Your password will expire in
+oie.password.expiry.warning.time = days
```

Comment:
got to be in one key ? `Your password will expire in {0} days`

Reply:
🤦